omnibus/resources/agent/msi/source.wxs.erb

```diff
@@ -673,8 +597,33 @@
     <WixVariable Id="WixUIDialogBmp" Value="Resources\assets\dialog_background.bmp" />
     <WixVariable Id="WixUIBannerBmp" Value="Resources\assets\banner_background.bmp" />
 
-    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall' 
-        Value='PROJECTLOCATION=[PROJECTLOCATION];APPLICATIONDATADIRECTORY=[APPLICATIONDATADIRECTORY];DDAGENTUSER_NAME=[DDAGENTUSER_NAME];DDAGENTUSER_PASSWORD=[DDAGENTUSER_PASSWORD];SYSPROBE_PRESENT=[SYSPROBE_PRESENT];ADDLOCAL=[ADDLOCAL]' />
+    <CustomAction Id='SetFinalizeProperty' Return='check' Property='FinalizeInstall'
+        Value='
+            PROJECTLOCATION=[PROJECTLOCATION];
+            APPLICATIONDATADIRECTORY=[APPLICATIONDATADIRECTORY];
+            DDAGENTUSER_NAME=[DDAGENTUSER_NAME];
+            DDAGENTUSER_PASSWORD=[DDAGENTUSER_PASSWORD];
+            SYSPROBE_PRESENT=[SYSPROBE_PRESENT];
+            ADDLOCAL=[ADDLOCAL];
+            APIKEY=[APIKEY];
+            TAGS=[TAGS];
+            HOSTNAME=[HOSTNAME];
+            PROXY_HOST=[PROXY_HOST];
+            PROXY_PORT=[PROXY_PORT];
+            PROXY_USER=[PROXY_USER];
+            PROXY_PASSWORD=[PROXY_PASSWORD];
+            LOGS_ENABLED=[LOGS_ENABLED];
+            APM_ENABLED=[APM_ENABLED];
+            PROCESS_ENABLED=[PROCESS_ENABLED];
+            CMD_PORT=[CMD_PORT];
+            SITE=[SITE];
+            DD_URL=[DD_URL];
+            LOGS_DD_URL=[LOGS_DD_URL];
+            PROCESS_DD_URL=[PROCESS_DD_URL];
+            TRACE_DD_URL=[TRACE_DD_URL];
+            PYVER=[PYVER];
+            HOSTNAME_FQDN_ENABLED=[HOSTNAME_FQDN_ENABLED];
+            ' />
```

Comment:
It won't be obvious for someone adding a new property that the last `;` and the last newline are mandatory. A way to solve it would be to do a simple string split instead of using a regex: first remove all `;` from here, then in C++ split the string based on `\n` separator, for each line trim whitespace on both ends, discard empty lines. We can do this on a separate PR if you prefer, but in that case please add a comment here explaining that both the `;` and newline at the end are mandatory.

Reply:
Yes, definitely the `;` is redundant now that we have `\n`. My point was that we could go even further and not require the last `\n` by using a simple string split by `\n` instead of a regex.

Reply:
We can do that change on a separate PR, though, but in that case I would add a comment mentioning that the last newline is important.